Makefile

```diff
@@ -38,6 +38,7 @@ GO111MODULE = on
 # ====================================================================================
 # Setup Kubernetes tools
 
+HELM_VERSION=v2.17.0
```

Comment:
do we still need to update the build submodule if we are using this newer version of helm2?

Reply:
We don't, but I think it is good to continue to update frequently. In reality I think we should have the build submodule versioned, but right now the latest commit it essentially the latest version.